### ISSUE TITLE
comment-no-empty should ignore /* stylelint-disable-next-line ... */

### ISSUE THREAD

**Issue by ilyub** [stylelint/stylelint#5422]
Sample code:
```css
/* stylelint-disable-next-line declaration-no-important */
color: red !important;
```

Sample config:
```js
{
  "rules": {
    "scss/comment-no-empty": true
  }
}
```

> What did you expect to happen?

The rule should ignore stylelint own disable directives

**Comment by jeddy3** [comment]
Thanks for the report.

I'm unable to reproduce this [using the demo](N4Igxg9gJgpiBcID0AqABAZwC4E8A2MeAlgHZYC0URGAhgEYHkkwAeFxzasYeNATjSxEIJJhHJEAtgAcIfLDTJoUSADolIeOfDR8YUNAEIps) with the built-in [`comment-no-empty` rule](https://stylelint.io/user-guide/rules/list/comment-no-empty/).

The issue may be with the [3rd party rule in the stylelint-scss plugin](https://github.com/kristerkari/stylelint-scss/tree/master/src/rules/comment-no-empty). I suggest creating an issue in their tracker.

Closing for now. You're welcome to reopen this issue with a reproducible example of the built-in rule not working, though.
